src/kubectlUtils.ts

```diff
@@ -459,6 +460,19 @@ export async function namespaceResources(kubectl: Kubectl, ns: string): Promise<
     );
 }
 
+export async function clusterResources(kubectl: Kubectl, verbs: string[]): Promise<Errorable<ResourceKind[]>> {
+    const arresult = await kubectl.readTable('api-resources -o wide');
+    if (ExecResult.failed(arresult)) {
+        return { succeeded: false, error: [ExecResult.failureMessage(arresult, {})] };
+    }
+
+    const resourceKinds: ResourceKind[] = arresult.result.filter((r) => verbs.every((verb) => r.verbs.includes(verb)))
+                                         .map((r) =>
+                                            new ResourceKind(r.name, r.name, r.kind, r.shortnames === '' ? r.kind : r.shortnames));

```

Comment:
This refers to 'shortnames' plural.  If a thing can have more than one shortname, we should be a bit careful about what we populate here.  It may not be an issue for the current use case, but once this function goes in, people will use it for all sorts of things and won't know about any potential shortname issues.  Mark my words...!

Reply:
I am still unsure how to handle this. I replaced the shortname with the name. I guess this should solve future potential issue.. ???